misc/main-api.cpp

```diff
@@ -191,6 +191,8 @@ void read_and_set_co2(double tstrt, double tend, Core &core, istream &sim_gcam_e
                              message_data(t, unitval(daccs, U_PGC_YR)));
             core.sendMessage(M_SETDATA, D_LUC_EMISSIONS,
                              message_data(t, unitval(luc, U_PGC_YR)));
+            core.sendMessage(M_SETDATA, D_LUC_UPTAKE,
+                             message_data(t, unitval(luc, U_PGC_YR)));
```

Comment:
Not related to this work but all of the inputs need to be added to the misc/main-api?

Reply:
Yes, they will. I added this mostly as a "don't forget" line (I've never built the GCAM API target myself).